Project.toml

```diff
@@ -0,0 +1,20 @@
+name = "PortAudio"
+uuid = "80ea8bcb-4634-5cb3-8ee8-a132660d1d2d"
+repo = "https://github.com/JuliaAudio/PortAudio.jl.git"
+version = "1.1.0"
+
+[deps]
+Libdl = "8f399da3-3557-5675-b5ff-fb832c97cbdb"
+LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
+SampledSignals = "bd7594eb-a658-542f-9e75-4c4d8908c167"
+libportaudio_jll = "2d7b7beb-0762-5160-978e-1ab83a1e8a31"
```

Comment:
I think you will need to add `FFTW` as a dependency here.

Reply:
I don't understand - I don't think I'm using FFTW (at least not intentionally)

Reply:
Oh, sorry, that was in `examples/`.  Nevermind.  :P